.github/workflows/cla.yml

```diff
@@ -0,0 +1,23 @@
+name: "CLA Signature Bot"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+jobs:
+  CLABot:
+    if: github.event_name == 'pull_request_target' || contains(github.event.comment.html_url, '/pull/')
```

Comment:
Can you explain what the second boolean expression is checking?

Reply:
It's checking to ensure that the URL for the comment includes the path segment `/pull/`, which it does for _pull reuest_ comments but not issue comments.